1.py

```diff
@@ -0,0 +1,8 @@
+# https://projecteuler.net/problem=1
+
+def three_or_five_multiple(until):
+    for num in range(1, until):
+        if num % 3 == 0 or num % 5 == 0:
+            yield num

```

Comment:
im guessing `yeild` adds this to a list instead of just returning it?

Reply:
https://wiki.python.org/moin/Generators